README.md

```diff
@@ -144,10 +144,10 @@ This is Day to Day Work Linux Cheatsheet for Software Engineers.
 | Name   |   Command |
 |------------ | -------------|
 |Stop Current Command| CTRL + c
-|Send Current Progress to Background| CTRL + z
-|Recall Last Command that match with given character| CTRL + r
-|Recall Latest Command| !!
-|Recall Latest Command that Start with Given String| !`<string>`
+|Sleep Program| CTRL + z <correct>
+|Search History| CTRL + r <correct>
+|Repeat Last Command| !! <correct>
+|Run Last Command Start with Given character| ! <character>
```

Comment:
For line "150", I think it should be !! <string>?

The above lines are OK.

|Sleep Program| CTRL + z
|Search History| CTRL + r
|Repeat Last Command| !!

Reply:
|Sleep Program| CTRL + z
|Search History| CTRL + r
|Repeat Last Command| !!
|Run Last Command Start with Given String| !! `<string>`

Reply:
![image](https://user-images.githubusercontent.com/52349645/94909172-ff6b6580-04cc-11eb-9421-be2ac1751c14.png)
I use only one ! and I works.

Reply:
![image](https://user-images.githubusercontent.com/52349645/94909251-19a54380-04cd-11eb-9b82-c0f68162a923.png)
If use !! `<string>` It will be like this.

Reply:
OK, I got it, so these should be as below:

|Sleep the current running process | CTRL + z
|Search commands from history | CTRL + r
|Repeat the latest command | !!
|Run the most recent command that matches with starting character(s) of string | !`<string>`